src/qt/clientmodel.h

```diff
@@ -105,13 +105,13 @@ class ClientModel : public QObject
     //! A thread to interact with m_node asynchronously
     QThread* const m_thread;
 
-    void TipChanged(SynchronizationState sync_state, interfaces::BlockTip tip, double verification_progress, bool header) EXCLUSIVE_LOCKS_REQUIRED(!m_cached_tip_mutex);
+    void TipChanged(SynchronizationState sync_state, interfaces::BlockTip tip, double verification_progress, int header) EXCLUSIVE_LOCKS_REQUIRED(!m_cached_tip_mutex);
```

Comment:
As a follow up, instead of this function the following two ones could be introduced:
- `BlockTipChanged(sync_state, tip, verification_progress)`
- `HeaderTipChanged(sync_state, tip, presync)`

Implementation each of them will be much simpler than the current `TipChanged`'s implementation.